### PR TITLE
Decreases npm install time ~25 seconds.

### PR DESCRIPTION
**Pull request by KaffinPX**
If you dont pull this, you will lose it lol

### CODE REVIEW

**Comment by rom1504** [comment]
npm install before: https://github.com/PrismarineJS/flying-squid/runs/2499415198 17s
npm install after https://github.com/PrismarineJS/flying-squid/pull/498/checks?check_run_id=2586175785 18s
so this PR is not improving the npm install time

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ world/*
 !world/.gitkeep
 package-lock.json
 /README.md
+config/default-settings.json
```
Comment (rom1504) [inline .gitignore:13]: that's a bug

File: config/default-settings.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "motd": "A Minecraft Server \nRunning flying-squid",
   "port": 25565,
   "max-players": 10,
-  "online-mode": true,
+  "online-mode": true ,
```
Comment (rom1504) [inline config/default-settings.json:5]: why did you change this ?

File: package.json [hunk 3]
```diff
@@ -35,8 +35,6 @@
     "minecraft-data": "^2.63.0",
     "minecraft-protocol": "^1.15.0",
     "mkdirp": "^0.5.1",
-    "moment": "^2.10.6",
-    "needle": "^2.5.0",
```
Comment (rom1504) [inline package.json:39]: why did you remove needle ?

File: src/lib/plugins/log.js [hunk 4]
```diff
@@ -23,13 +22,21 @@ module.exports.server = function (serv, settings) {
...
 
-  serv.on('seed', (seed) => serv.log('seed: ' + seed))
+  serv.on('seed', (seed) => serv.log('Seed: ' + seed))
 
   const logFile = path.join('logs', timeStarted + '.log')
 
   serv.log = message => {
     readline.cursorTo(process.stdout, 0)
-    message = moment().format('MMMM Do YYYY, HH:mm:ss') + ' ' + message
+    const date = new Date()
```
Comment (rom1504) [inline src/lib/plugins/log.js:31]: why did you replace moment by this fragile code ?

File: src/lib/plugins/moderation.js [hunk 5]
```diff
@@ -1,12 +1,11 @@
...
-const needle = require('needle')
+const https = require('https')
 const UserError = require('flying-squid').UserError
 
 module.exports.server = function (serv, settings) {
   serv.ban = async (uuid, reason) => {
     if (!serv.bannedPlayers[uuid]) {
       serv.bannedPlayers[uuid] = {
-        time: +moment(),
+        time: +(Date.now() / 1000 + '').replace(/\./, ''),
```
Comment (rom1504) [inline src/lib/plugins/moderation.js:8]: this looks pretty bad

File: src/lib/plugins/moderation.js [hunk 6]
```diff
@@ -30,15 +29,26 @@ module.exports.server = function (serv, settings) {
...
   serv.getUUIDFromUsername = async username => {
-    return await new Promise((resolve, reject) => {
-      needle('get', 'https://api.mojang.com/users/profiles/minecraft/' + username, { json: true })
-        .then((response) => {
-          if (!response.body) throw new Error('username not found')
-          const idstr = response.body.id
-          if (typeof idstr !== 'string') throw new Error('username not found')
-          resolve(uuidInParts(idstr))
+    return new Promise(function (resolve, reject) {
+      const req = https.request({
```
Comment (rom1504) [inline src/lib/plugins/moderation.js:33]: this is much worse than before

**Comment by rom1504** [comment]
reviewed.
Do you have any particular objective here?
Removing dependencies is not useful in itself.

File: src/lib/plugins/moderation.js (hunk 5 above)
Comment (Karang) [reply src/lib/plugins/moderation.js:8]: I don't think he realize this code is equivalent to just `Date.now()` ?

**Comment by KaffinPX** [comment]
ur lose lol

File: src/lib/plugins/log.js (hunk 4 above)
Comment (KaffinPX) [reply src/lib/plugins/log.js:31]: moment = heavy, slow
